Mapping Tools/views/Sliderator/SlideratorView.xaml.cs

```diff
@@ -77,20 +77,23 @@ public partial class SlideratorView {
         private void Import_Slider(object sender, RoutedEventArgs e)
         {
             bool editorRead = EditorReaderStuff.TryGetFullEditorReader(out var reader);
-                foreach (string path in MainWindow.AppWindow.GetCurrentMaps())
-                {
-                    var selected = new List<HitObject>();
-                    BeatmapEditor editor = editorRead ? EditorReaderStuff.GetNewestVersion(path, out selected, reader) : new BeatmapEditor(path);
-                    Beatmap beatmap = editor.Beatmap;
-                    Timing timing = beatmap.BeatmapTiming;
-                    List<HitObject> markedObjects = selected;
-                    
+            foreach (string path in MainWindow.AppWindow.GetCurrentMaps())
+            {
+                var selected = new List<HitObject>();
+                HitObject CurrentViewed = GraphHitObjectElement.HitObject;
+                BeatmapEditor editor = editorRead ? EditorReaderStuff.GetNewestVersion(path, out selected, reader) : new BeatmapEditor(path);
+                Beatmap beatmap = editor.Beatmap;
+                Timing timing = beatmap.BeatmapTiming;
+                List<HitObject> markedObjects = selected;
+
+                try {
                     GraphHitObjectElement.HitObject = markedObjects.Find(s => s.IsSlider);
+                } catch (ArgumentNullException) { }
```

Comment:
might have been wrong on this one. I think you will never get an ArgumentNullException because the argument is always `s => s.IsSlider`. It will probably return null if it can't find the slider so use `var selectedSlider = markedObjects.Find(s => s.IsSlider); if(selectedSlider != null) {GraphHitObjectElement.HitObject = selectedSlider}`

Reply:
It actually returns the Default Hitobject object.
![image](https://user-images.githubusercontent.com/37494321/70849222-1ea9ef80-1ec7-11ea-9e24-401e1d3a3276.png)

Reply:
Default value could be a null because i never specified a default value for HitObject. I suggest using `markedObjects.First(s => s.IsSlider)` and then catching the InvalidOperationException. I know for sure `First` throws a InvalidOperationException if it can't find the element.

Reply:
First isn't apart of the List of Hitobjects.. I think you got yourself confused with the IEnumerable's again. But will check for null.

Reply:
A List is a IEnumerable so it can use First from linq. Do you use Find because of some performance reason?

Reply:
Well. I just used it because I forgot to import linq.. It's the same thing